src/components/SearchBox.vue

```diff
@@ -33,6 +37,9 @@
     },
     created() {
       this.searchQuery = this.queryDisplay;
+      this.MeloarInstances.instances.filter(item => {
```

Comment:
I see what you are doing here but it is hard to read.

Reply:
Refactor hard, or comment hard? :o

Reply:
We will merge this one as an example of the slickness we need to harness so that developers after us can easily read the code :))